app/models/live_issue.rb

```diff
@@ -0,0 +1,12 @@
+class LiveIssue < Sequel::Model(Sequel[:live_issues].qualify(:public))
+  plugin :timestamps, update_on_create: true
+  plugin :auto_validations, not_null: :presence
+
+  def validate
+    super
+
+    validates_presence %i[title status commodities date_discovered]
+    validates_max_length 256, %i[description suggested_action]
+    validates_includes %w[Active Resolved], :status
```

Comment:
This is something I'd like to discuss with someone. I've added status like this because in the future we're looking into including 'Archived'. Now I'm thinking that maybe it should a boolean on its own so you can validate if it's resolved before you toggle Archived on. What do people think?